docs/sdk/examples/angular.html

```diff
@@ -233,91 +203,102 @@ <h1>CKEditor 4 Angular Integration <a class="documentation" href="https://ckedit
 	selector: 'app-events',
 	template: `
-		<h2>Events and Directives</h2>
+	<h2>WYSIWYG editor with custom event handlers and configuration</h2>
 
-		<p>
-			The CKEditor 4 WYSIWYG editor component allows you to bind event handlers with the <a href="https://ckeditor.com/docs/ckeditor4/latest/guide/dev_angular.html#supported-output-properties"><code>@Output</code></a> attributes in the HTML code.
-		</p>
+	<p>
+		Editors created with the CKEditior 4 Angular component are highly customizable. It is possible to overwrite every configuration setting using the <code>config</code> directive and passing an object containing the configuration to it.
+	</p>
 
-		<p>
-			The following example shows how to bind several CKEditor 4 events. Additionally, it presents how the component could be used with the <a href="https://angular.io/guide/attribute-directives">Angular directives</a>.
-		</p>
+	<p>
+		Additionally, the CKEditor 4 WYSIWYG editor component for Angular allows you to bind predefined component events using <a href="https://ckeditor.com/docs/ckeditor4/latest/guide/dev_angular.html#supported-output-properties"><code>@Output</code></a> attributes. The following example shows how to bind several common CKEditor 4 events.
+	</p>
 
-		<ckeditor
-			*ngIf="!isRemoved"
-			data="<p>This is a CKEditor 4 instance created with Angular.</p>"
-			[readOnly]="isReadOnly"
-			[hidden]="isHidden"
-
-			(ready)="onReady()"
-			(change)="onChange()"
-			(focus)="onFocus()"
-			(blur)="onBlur()">
-		</ckeditor>
+	<ckeditor
+		data="<p>This is a CKEditor 4 instance created with Angular.</p>"
 
-		<h3>Events log</h3>
+		[(config)]="editorConfig"
 
-		<ul class="events-log">
-			<li *ngFor="let event of componentEvents">{{ event.message }}>
-				<span *ngIf="event.counter > 1"> ({{ event.counter }})</span>
-			</li>
-		</ul>
+		(ready)="onReady()"
+		(change)="onChange()"
+		(focus)="onFocus()"
+		(blur)="onBlur()">
+	</ckeditor>
 
+	<section>
+		<h3>Events log</h3>
+		<small>To check additional details about every event, consult the console in the browser developer tools.</small>
+		<div class="events__log">
+			<ul>
+				<li *ngFor="let event of events">
+					{{ event.timestamp }} - {{ event.name }}
+				</li>
+			</ul>
+		</div>
 		<button (click)="clearEventsLog()">Clear events log</button>
-
-		<h3>Angular directives</h3>
-
-		<p><label><input type="checkbox" [(ngModel)]="isHidden">Hide the editor with the <code>[hidden]</code> directive.</label></p>
-		<p><label><input type="checkbox" [(ngModel)]="isRemoved">Remove the editor from the DOM with the <code>*ngIf</code> directive.</label></p>
-		<p><label><input type="checkbox" [(ngModel)]="isReadOnly">Make the editor read-only.</label></p>
+	</section>
 	`
 } )
 export class EventsComponent {
-	public isReadOnly = false;
-	public isHidden = false;
-	public isRemoved = false;
-	public componentEvents: ComponentEvent[] = [];
+	public events: ComponentEvent[] = [];
+
+	public editorConfig = {
+		toolbar: [
```

Comment:
Please remove the custom toolbar config from the `Configuration` samples for all frameworks as you did it for the `Two-way Data Binding` ones. Also code snippets need to be adjusted afterwards.

Reply:
This sample shows how to apply a custom config for the editor, that's the reason for the bookmark name `Configuration`.

Reply:
Right, it makes sense 🙈